### DISCUSSION TITLE
Grafana Image Renderer plugin pre-installed

### DISCUSSION THREAD

**Discussion by rschitz** [teslamate-org/teslamate#2717] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Hi, thanks a lot for this awesome tool.
It would be nice to have the Grafana Image Renderer plugin pre-installed to be able to share graph without having to screenshot them.
https://grafana.com/docs/grafana/latest/installation/docker/
GF_INSTALL_IMAGE_RENDERER_PLUGIN=true 

### Relevant log output

```shell
N/A
```

### Type of installation

Docker

### Version

v1.26.1

**Comment by adriankumpf** [comment]
According to https://grafana.com/grafana/plugins/grafana-image-renderer/ this requires a whole chromium instance to be added to the Docker image which would increase the image size significantly. I have doubts whether it would be worth it. Especially if only a small part of the users would use this feature.
